ext/pdo_oci/tests/pdo_oci_stream_2.phpt

```diff
@@ -69,6 +69,57 @@ printf("Done\n");
 /* Cleanup is done in pdo_oci_stream_2b.phpt */
 //$db->exec("drop table pdo_oci_stream_2");
 
+$db->setAttribute(PDO::ATTR_STRINGIFY_FETCHES, false);  // Let's use streams
```

Comment:
why the `PDO::ATTR_STRINGIFY_FETCHES = true` case had to be removed?

Reply:
It is not removed, it is moved from https://github.com/php/php-src/pull/9048/files#diff-e8760b378cb0d846f88a694ac03bba4526f34aa3e59db649e59e68e26e2f5375L18

the `ext/pdo_oci/tests/pdo_oci_stream_2a.phpt` and `ext/pdo_oci/tests/pdo_oci_stream_2b.phpt` tests are moved together, as the b depends on a, and the b test was failing randomly when tests are run in parallel